tests/security_and_privacy/test_phishing_and_malware_warnings.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+from modules.browser_object_tabbar import TabBar
+
+
+@pytest.fixture()
+def add_prefs():
+    return [
+        ("browser.safebrowsing.malware.enabled", True),
+        ("browser.safebrowsing.phishing.enabled", True),
+    ]
+
+
+def get_error_title(driver, url, expected_title):
+    """
+    Navigate to the given URL and assert the error title text.
+    """
+    driver.get(url)
+    error_title_element = WebDriverWait(driver, 10).until(
+        EC.presence_of_element_located((By.ID, "errorTitleText"))
+    )
+    error_title = error_title_element.get_attribute("innerText")
+    assert error_title == expected_title, (
```

Comment:
We should probably refrain from performing the assertion inside helper functions. This function should probably just return the error_ttile.

Reply:
Ok, good to know. I've moved the assertion inside the actual test.